packages/grid/helm/helm.py

```diff
@@ -103,6 +105,61 @@ def ingress_with_tls() -> str:
         return fp.read()
 
 
+def add_notes(helm_chart_template_dir: str) -> None:
+    """Add notes or information post helm install or upgrade."""
+
+    notes = """
+    Thank you for installing {{ .Chart.Name }}.
+    Your release is named {{ .Release.Name }}.
+    To learn more about the release, try:
+
+        $ helm status {{ .Release.Name }}
```

Comment:
MAYBE: we could add `-n {{ .Release.Namespace }}`  at the end as the above command would not work normally

Reply:
Resolved by [b141478](https://github.com/OpenMined/PySyft/pull/8202/commits/b141478956387e69f8cc53e97485a9492b6b7168)